public/kibi_timeline.js

```diff
@@ -80,6 +81,19 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.substring(0,7) === 'Etc/GMT') {
```

Comment:
Can you use [startsWith](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) for this condition ?